### PR TITLE
EventForm component

### PR DESCRIPTION
**Pull request by Devocolypse**
As of writing, event form and request works as expected: information is entered, required or otherwise, and a proper request is made to the back end.

### Caveats
In order for the axios request to play nice with what's already set up in the back end, I had to _pretend_ to be logged in as a user. I did this by manually storing a localStorage token in the browser console, which I generated using Postman requests to the back end (i.e. POST /api/user/login), which spat out a JWT. Then, I modified the front end's EventContext to retrieve this token and then set it as the Authorization header.

**TL;DR** - Creating a new event works, provided there is an authenticated user doing so.

### CODE REVIEW

**Review by RobbieMac82 — APPROVED** [review]
Great Job!
